atest/robot/tags/tag_stat_include_and_exclude.robot

```diff
@@ -5,6 +5,7 @@ Resource          atest_resource.robot
 *** Variables ***
 ${DATA SOURCE}    tags/include_and_exclude.robot
 ${F}              force
+${INT}            robot:foobar
```

Comment:
`${INT}` is a bit strange variable name because it sounds like `int` i.e. integer. It seems to be used as an example of an internal tag so perhaps `${INTERNAL}` would be better.

The value could also be changed to `robot:just-an-example` to make it a bit more explicit as well. I first though this was an example of an invalid value but it seems to be just an example.